Dockerfile

```
@@ -10,7 +10,7 @@ ARG LIBXEXT_VERSION
 ARG MINICONDA_VERSION=py310_23.1.0-1
 ARG MAKE_VERSION=4.2.1-1.2
 ARG GIT_VERSION=1:2.25.1-*
-ARG WGET_VERSION=1.20.3-1ubuntu2
+ARG WGET_VERSION=1.20.3-1ubuntu2.1
```

Might be better to do
```suggestion
ARG WGET_VERSION=1.20.3-*
```

These ubuntu specific versions seem to disappear over time